### PR TITLE
PR: Changed package and class names to adhere to the Java naming conventions

### PR DESCRIPTION
**Pull request by ghost**
Packages, such as `commandHandling`, `adminCommands`, etc., have been changed to adhere to the naming conventions of Java. Furthermore, some package names have been shortened to keep everything concise and clean :3

### CODE REVIEW

File: src/main/java/commandhandling/commands/pubIic/SpokesPeople.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
-package commandHandling.commands.publicCommands;
+package commandhandling.commands.pubIic;
```
Comment (4ndriH) [inline src/main/java/commandhandling/commands/pubIic/SpokesPeople.java]: typo here, `pubIic` is not a correct

**Review by 4ndriH — CHANGES_REQUESTED** [review]
General typo in `public` commands package name

**Comment by ghost** [comment]
@4ndriH Pushed your requested changes :3

**Review by 4ndriH — APPROVED** [review]
thank you for implementing the changes

**Comment by 4ndriH** [comment]
merging it to a separate branch for further changes

**Pull request merged**